interpreter/misc_test.go

```diff
@@ -13480,7 +13476,13 @@ func TestInterpretVariableDeclarationSecondValueEvaluationOrder(t *testing.T) {
 				},
 			),
 			"",
-			func(invocation interpreter.Invocation) interpreter.Value {
+			func(
+				context interpreter.NativeFunctionContext,
```

Comment:
Here and elsewhere the parameter names are unused, they can be replaced with `_`